atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/timestamp/InDbTimestampBoundStore.java

```diff
@@ -45,15 +50,23 @@
     private static final Logger log = LoggerFactory.getLogger(InDbTimestampBoundStore.class);
 
     private final ConnectionManager connManager;
+    private final TableReference timestampTable;
 
     @GuardedBy("this") // lazy init to avoid db connections in constructors
     private DBType dbType;
 
     @GuardedBy("this")
     private Long currentLimit = null;
 
-    public InDbTimestampBoundStore(ConnectionManager connManager) {
+    public static InDbTimestampBoundStore create(ConnectionConfig connConfig) {
+        return new InDbTimestampBoundStore(
+                new HikariCPConnectionManager(connConfig, Visitors.emptyVisitor()),
+                AtlasDbConstants.TIMESTAMP_TABLE);
+    }
+
+    public InDbTimestampBoundStore(ConnectionManager connManager, TableReference timestampTable) {
         this.connManager = Preconditions.checkNotNull(connManager);
+        this.timestampTable = Preconditions.checkNotNull(timestampTable);
```

Comment:
It seems a bit odd to have this considered a part of KVS init process when really it is not used by the KVS.  I imagine that this should become part of a separate set of timestamp constants and ignored by the KVS, but I need to think about this a bit more.